hbase-shell/src/test/ruby/shell/rsgroup_shell_test.rb

```diff
@@ -127,5 +127,17 @@ def setup
         @hbase.rsgroup_admin().get_rsgroup_of_table('foobar')
       end
     end
+
+    define_test 'Test rsgroup rename' do
+      old_rs_group_name = 'test_group'
+      new_rs_group_name = 'renamed_test_group'
+      table_name = 'test_table'
+
+      @hbase.rsgroup_admin.rename_rsgroup(old_rs_group_name, new_rs_group_name)
+      assert_not_nil(@admin.getRSGroup(new_rs_group_name))
+      assert_equal(1, @admin.getRSGroup(new_rs_group_name).getServers.count)
+      assert_equal(1, @admin.listTablesInRSGroup(new_rs_group_name).count)
+      assert_true(@admin.listTablesInRSGroup(new_rs_group_name).contains(org.apache.hadoop.hbase.TableName.valueOf(table_name)))
```

Comment:
Does it make sense to also check if the old rsgroup is gone?

Reply:
Add that check in the new commit.